### PR TITLE
Yarn commands for dev/formatting 🤠

### PR DESCRIPTION
**Pull request by Leundai**
## Description :star2:

image

image

You need to install concurrently!
`npm install -g concurrently`

### CODE REVIEW

**Comment by chyku** [comment]
i appreciate this but this is lower priority than our feature/bug issues :sob:

File: backend/Makefile [hunk 1]
```diff
@@ -38,4 +38,12 @@ make compose_start:
 
 heroku_setup:
 	python manage.py recreate_db
-	
\ No newline at end of file
+	
+format:
```
Comment (chyku) [inline backend/Makefile:42]: LMAO good on u for finding this file, if you couldn't tell it's not updated from the original code we repurposed for mongodb

**Review by chyku — CHANGES_REQUESTED** [review]
pls add documentation to a readme somewhere that's all i request

**Review by chyku — APPROVED** [review]

**Review by AngelaLuo49021 — APPROVED** [review]
kool

**Pull request merged**